zerver/webhooks/patreon/doc.md

```diff
@@ -1,22 +1,35 @@
+# Zulip Patreon integration
+
 Get Patreon notifications in Zulip!
 
- 1. {!create-channel.md!}
+{start_tabs}
+
+1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-    You can refer to Patreons's documentation for
-    [triggers](https://docs.patreon.com/#triggers-v2) and
-    [webhook events](https://docs.patreon.com/#webhooks).
 
 1. Login to your Patreon developer account, navigate to your Patreon Portal, and
-   click on [My Webhooks](https://www.patreon.com/portal/registration/register-webhooks).
+    click on [My Webhooks](https://www.patreon.com/portal/registration/register-webhooks).
```

Comment:
Use a reference link instead?

Reply:
Not sure what's a reference link. Do you mean the related documentation section?

Reply:
I meant this.
```
[My Webhooks][1]

[1]: URL
```
Put the ref links at the bottom of the document.